frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -306,18 +306,21 @@ export const constructCurrentPipeline = (
   }
 
   const latestRun = getLatestRun({ data: pipelineRuns }, 'creationTimestamp');
-
+  let status: string;
   if (!latestRun) {
-    // Without the latestRun we will not have progress to show
-    return null;
+    status = runStatus.PipelineNotStarted;
```

Comment:
Nit: You kept the exit early, this could have just been added to the return directly and no other changes needed to be done.